app/assets/javascripts/discourse/ember-cli-build.js

```diff
@@ -191,10 +191,9 @@ module.exports = function (defaults) {
   const terserPlugin = app.project.findAddonByName("ember-cli-terser");
   const applyTerser = (tree) => terserPlugin.postprocessTree("all", tree);
 
-  return mergeTrees([
+  return app.toTree([
```

Comment:
It looks like passing our custom trees in here will cause them to be run through any addons which have a `postprocessTree` hook. Main one I'm thinking of [is terser](https://github.com/ember-cli/ember-cli-terser/blob/master/index.js#L26-L30).

So I guess we'll need to remove our custom `applyTerser()` wrapper from all our trees to avoid double-minifying code in production builds

Reply:
Good call! (It was probably already doing that in the embroider build)

This actually seems like an improvement overall, but do you think there is anything we need to tweak in the terser settings? In particular the `exclude` option? Since this now cover a few more trees that were previously not minified (not sure how intentional that was)

Reply:
Should be fine. A few more files will be minified, but I agree that's likely to be an improvement. I'll keep an eye on production build times just to be sure there's no large regression.